Cron/ProcessPendingOrder.php

```diff
@@ -38,32 +45,34 @@ public function __construct(
      * @throws PlacetoPayException
      * @throws LocalizedException
      */
-    public function execute()
+    public function execute(): void
     {
         /** @var Order $orders */
         $orders = $this->collectionFactory->create()
             ->addAttributeToSelect('*')
             ->addAttributeToFilter('state', ['in' => [
-                Order::STATE_PENDING_PAYMENT,
-                Order::STATE_NEW,
+              Order::STATE_PENDING_PAYMENT,
+              Order::STATE_NEW,
             ]])
             ->addAttributeToFilter('status', ['in' => [
-                'pending_payment',
-            ]]);
+                PaymentStatus::PENDING_PAYMENT, PaymentStatus::PENDING
+            ]])->addAttributeToSort('entity_id');
 
         if ($orders) {
             foreach ($orders as $order) {
-                $requestId = $order->getPayment()->getAdditionalInformation()['request_id'];
-
-                if (!$requestId) {
-                    continue;
+                $this->logger->debug('Process order pending' . $order->getRealOrderId());
+                $information = $order->getPayment()->getAdditionalInformation();
+                if (empty($information['request_id'])) {
+                    $this->placetopay->processPendingOrderFail($order);
+                } else {
```

Comment:
To reduce cognitive load, remove else and use `continue` in if conditional to preserve flow